ARGUS/ARGUS_UNet/utils/ARGUS_vfold_setup.py

```diff
@@ -0,0 +1,169 @@
+import os
+import numpy as np
+from glob import glob
+from monai.data import CacheDataset, DataLoader
+import ubelt as ub
+import pint
+Ureg = pint.UnitRegistry()
+
+def setup_ONSD_vfold_files(self, img_dir, anno_dir, p_prefix, n_prefix):
+    self.all_train_images = sorted(glob(os.path.join(img_dir, "*_cropM.mha")))
+    self.all_train_labels = sorted(glob(os.path.join(anno_dir, "*.overlay.mha")))
+
+    total_bytes = 0
+    for p in self.all_train_images:
+        p = ub.Path(p)
+        total_bytes += p.stat().st_size
+    print("Total size of images in the dataset: ")
+    print((total_bytes * Ureg.byte).to("GiB"))
+
+    total_bytes = 0
+    for p in self.all_train_labels:
+        p = ub.Path(p)
+        total_bytes += p.stat().st_size
+    print("Total size of labels in the dataset: ")    
+    print((total_bytes * Ureg.byte).to("GiB"))
+
+    num_images = len(self.all_train_images)
+    print("Num images / labels =", num_images, len(self.all_train_labels))
+
+    fold_prefix_list = []
+    p_count = 0
+    n_count = 0
+    for i in range(self.num_folds):
+        num_p = 1
+        num_n = 1
+        if i > self.num_folds - 5:
```

Comment:
This is the one line that needs to be changed to make this generic.

Reply:
Are you referring to only line 36 here or lines 34-36?